source/documentation/administration_guide/chap-Hosts.adoc

```diff
@@ -72,6 +53,27 @@ include::topics/Accessing_Cockpit_from_the_Admin_Portal.adoc[]
 
 include::common/admin/proc-Setting_Legacy_SPICE_Cipher.adoc[leveloffset=+3]
```

Comment:
What was the reason to move above sections after Host PCI Passthrough and Legacy SPICE sections? Both those section seems to me as much less important than those which was moved further

Reply:
@mwperina  Because they didn't make logical sense in the toc. There's a section in the TOC called HOST TASKs, and in the middle of that is a bunch of topics explaining settings, and then some more Host Tasks.
I consolidated all the tasks together, and then moved the topics on settings after the tasks.